_includes/pbc/all/data-import-and-export/file-details-product-abstract.csv.md

```diff
@@ -48,16 +32,16 @@ The file should have the following parameters:
 | icecat_license |  | String | | Icecat product catalogue license code. |
 | new_from |  | Date | | To be considered a new product from this presented date. |
 | new_to |  | String | | To be considered a new product until this presented date. |
-| avalara_tax_code |  | String | | [Avalara tax code](/docs/scos/user/features/{{page.version}}/tax-feature-overview.html#avalara-system-for-automated-tax-compliance) for automated tax calculation. |
+| avalara_tax_code |  | String | | [Avalara tax code](/docs/pbc/all/tax-management/tax-feature-overview.html#avalara-system-for-automated-tax-compliance) for automated tax calculation. |
```

Comment:
We should only mention this part in the Avalara doc, not in the PIM default CSV import